.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,42 @@
+name: Release metrics adapter chart
+on:
+  push:
+    branches:
+      - main
+      - devel
+
+jobs:
+  # Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+  # This job checks the remote file is up to date with the local one on release
+#  validate-gh-pages-index:
```

Comment:
This has to be commented on the first run of the chart release because `index.yaml` in the `gh-pages` branch still does not exist and this step fails because it cannot compare an unexisting file with a 404 HTML file that is going to download.

If you create an empty `index.yaml` then the step that releases the chart is going to fail because `cr` doesn't like that `index.yaml` is empty and thinks that it found an error downloading the help repo index.

When we release our first chart a PR we can do a PR to uncomment this.

Reply:
was it tested in the fork? If so I am 100% fine with this approach :)

Reply:
It has a commit in this very PR when I commented it but Github has already rotated the logs where it failed.

Reply:
we could add manually the branch and an intex.yaml if we want to enable it since the beginning, but maybe it does not worth it

Reply:
I would have to test how much info must `index.yaml` have until `cr` works correctly. I would have to read `cr` code where it unmarshals the `index.yaml` or do a couple to pipeline runs.

I would prefer to do the first run and have everything to uncomment it. Yep. I does not worth it.